example/pubspec.lock

```diff
@@ -5,70 +5,70 @@ packages:
     dependency: transitive
     description:
       name: another_flushbar
-      url: "https://pub.dartlang.org"
+      url: "https://pub.flutter-io.cn"
```

Comment:
Please remove this file from your commit since it changes all the urls to Chinese ones.

Reply:
Ok.It's my wrong